datafusion/core/tests/sqllogictests/test_files/describe.slt

```diff
@@ -0,0 +1,43 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+
+#   http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+
+##########
+# Describe internal tables
+##########
+
+statement ok
+set datafusion.catalog.information_schema = true
+
+statement ok
+CREATE external table aggregate_simple(c1 real, c2 double, c3 boolean) STORED as CSV WITH HEADER ROW LOCATION 'tests/data/aggregate_simple.csv';
+
+query C1
+DESCRIBE aggregate_simple;
```

Comment:
This test works on master -- I don't think it needs the changes in this PR

```
(arrow_dev) alamb@MacBook-Pro-8:~/Software/arrow-datafusion/datafusion/core$ datafusion-cli
DataFusion CLI v16.0.0
❯ CREATE external table aggregate_simple(c1 real, c2 double, c3 boolean) STORED as CSV WITH HEADER ROW LOCATION 'tests/data/aggregate_simple.csv';
0 rows in set. Query took 0.005 seconds.
❯ 
DESCRIBE aggregate_simple;
+-------------+-----------+-------------+
| column_name | data_type | is_nullable |
+-------------+-----------+-------------+
| c1          | Float32   | NO          |
| c2          | Float64   | NO          |
| c3          | Boolean   | NO          |
+-------------+-----------+-------------+
3 rows in set. Query took 0.002 seconds.
❯ 
```

Reply:
there is no sqllogictests to test describe, so i add some tests here, now i unify the describe table and describe file, and i think it's necessary to add some tests to this.